src/main/java/uk/gov/justice/hmpps/prison/api/model/RequestToDischargePrisoner.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.hmpps.prison.api.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotNull
+import org.hibernate.validator.constraints.Length
+import org.springframework.format.annotation.DateTimeFormat
+import java.time.LocalDateTime
+
+@Schema(description = "Request release of prisoner")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class RequestToDischargePrisoner(
```

Comment:
Apart from conversion to Kotlin the only change to this request object is to drop the comment text - it isn't used anyway.